File: ex7plots.m
% ex7: Plots, MATLAB

foo = randn(10000, 1);

histogram(foo, 50, 'edgecolor', 'w');
xlabel('Bin');
ylabel('Value');
title('Histogram, Normal Distribution');
text(-3.5, 600, 'n=10,000, bins=50');
axis([-4, 4, 0, 800]);
grid('on');
shg();

File: ex7plots.py
# ex7: Plots, Python

import numpy as np
import matplotlib.pyplot as plt

foo = np.random.randn(10000, 1)

plt.hist(foo, 50, edgecolor='w')
plt.xlabel('Bin')
plt.ylabel('Value')
plt.title('Histogram, Normal Distribution')
plt.text(-3.5, 600, 'n=10,000, bins=50')
plt.axis([-4, 4, 0, 800])
plt.grid(True)
plt.show()
